pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -372,12 +381,18 @@ func (p *PriorityQueue) backoffPod(pod *v1.Pod) {
 	}
 }
 
+func (p *PriorityQueue) Cycle() int64 {
+	p.lock.Lock()
```

Comment:
I will recommend using p.lock.Lock() and p.lock.RUnlock() here. :)

Reply:
Thanks!